### DISCUSSION TITLE
Remnant: Keystone Research 1 - cloaked scanning?

### DISCUSSION THREAD

**Discussion by opusforlife2** [endless-sky/endless-sky#10632] [Lore]
Aeolus claims that cloaked ships have visited Hai space. But then he goes on to say that they have scanned Hai ships. However, we know the player cannot scan anything while being cloaked.

Is that just a game mechanic made to limit the player, but others are perfectly able to cloak and scan?

**Comment by Hecter94** [comment]
https://github.com/endless-sky/endless-sky/pull/10049

There is this, so if that PR is accepted, that may change in the future. 
Otherwise, it's probably not unreasonable that they've simply uncloaked around a lone Hai merchant or something a few times in the past to scan them while staying cloaked at all other times, and the scant few instances of it happening have gone largely unnoticed and unreported by the overall Hai government.

**Reply by opusforlife2** [reply]
Or, they wait in Wah Ki until every active ship has left, uncloak, and scan disabled ships.
